python/nav/web/api/v1/views.py

```diff
@@ -879,27 +878,26 @@ class AlertFragmentRenderer(TemplateHTMLRenderer):
     """
     media_type = 'text/x-nav-html'
 
-    def resolve_context(self, data, request, _response):
+    def get_template_context(self, data, renderer_context):
         """Populate the context used for rendering the template
 
-        :type request: rest_framework.request.Request
-        :type _response: rest_framework.request.Response
         :param dict data: The serialized alert
+        :param dict renderer_context: Existing context
         """
 
         if 'id' not in data:
-            return RequestContext(request, data)
+            return data
 
         # Put the alert object in the context
         data['alert'] = event.AlertHistory.objects.get(pk=data['id'])
 
-        netboxid = data.get('netbox')
+        netboxid = data.get('netbox', None)
```

Comment:
This seems redundant. The `get()` method will return `None` by default if there was no match.

Reply:
Fixed